hypershift-operator/controllers/hostedcluster/metrics.go

```diff
@@ -62,6 +63,7 @@ func init() {
 		limitedSupportEnabled,
 		silenceAlerts,
 		proxyConfig,
+		common.SkippedCloudResourcesDeletion,
```

Comment:
I don't think you can use the same metric in the HostedControlPlane operator if you just register it here.
you need to register it inside the `control-plane-operator`  folder so it is included in the binary and you would need to create a serviceMonitor/podMonitor for Prometheus to be able to scrape the metric from the controlPlaneOperator pod.
cc @enxebre correct me if I am wrong.

Reply:
We already have a podmonitor for the cpo, but you are correct in that we would need to register the metric in the cpo process. 
My preference would be to have only the HO report this metric if possible.

Reply:
The thing is, in the Jira ticket @enxebre mentioned also the CPO part, related mostly with the AWSDefaultSecurityGroup cleanup, so we have 2 opcions:

- I can register in the CPO the new metric and report it as I do in the HO part.
- Implement only the HO part and left the CPO unreported.

WDYT?

@csrwng @muraee

Reply:
As discussed in the [Slack thread](https://redhat-internal.slack.com/archives/G01QS0P2F6W/p1683624750678259) lets keep only the HO part for now.